src/jit/stacklevelsetter.cpp

```diff
@@ -151,6 +151,8 @@ void StackLevelSetter::SetThrowHelperBlocks(GenTree* node, BasicBlock* block)
             SetThrowHelperBlock(SCK_ARITH_EXCPN, block);
         }
         break;
+        default: // Other opers do not target throw blocks.
+            break;
```

Comment:
Is it correct to assume that we never gonna hit `default:` here?

Reply:
No, we hit it and ignore them. All opers that do not throw will hit default case.